python/create_mask.py

```diff
@@ -0,0 +1,138 @@
+"""
+
+; Project     : Coronal Hole Identification via Multi-thermal Emission Reconition Algorithm (CHIMERA)
+;
+; Name        : create_mask
+;
+; Purpose     : Create transparent CH masks from previously stored data
+;
+; Syntax      : create_mask.py path 
+;
+; Inputs      : CHIMERA ch_location.txt file
+;
+; Outputs     : Transparent CH mask
+;
+; Keywords    : path = directory path of file	
+;
+; Example     : ipython> % run -i create_mask.py '/path/to/file/location/'
+;
+; Required Packages    : pip install mahotas
+;
+; History     : v1.0, Written 14-Jun-2018, Tadhg Garton, TCD
+;
+; Contact     : gartont@tcd.ie
+;		info@solarmonitor.org
+
+"""
+
+from datetime import datetime
+import glob
+import sys
+
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+import os.path as op
+
+#=====Check python version=====
+
+if sys.version[0] != '3':
+    raise NameError('=====   Python version 3 must be used for create_mask.py   =====')
+
+#=====Establish pathing=====
+path = ''
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+    exis = op.exists(op.join(path,'meta/'))
+    if exis == 0:
```

Comment:
I'm pretty sure `exists` should be a bool.. Therefore if the file didn't exist you would be comparing `false == 0` which will give you `false` and miss this block.

You should do :
```python
if not exists:
   ...
```